modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/WikiPageAttachmentResourceImpl.java

```diff
@@ -118,6 +168,7 @@ private WikiPageAttachment _toWikiPageAttachment(FileEntry fileEntry)
 					"contentValue", fileEntry::getContentStream,
 					Optional.of(contextUriInfo));
 				encodingFormat = fileEntry.getMimeType();
+				externalReferenceCode = fileEntry.getExternalReferenceCode();
```

Comment:
Could we, in the future, move this code to a common one? I think this code is duplicated among the different entities' attachments code. Just at least take a look to see if it is possible easily or not.
cc/ @javierdearcos

Reply:
Good point. Can you create just s simple task with a little description to remember to look into this in the future, with the tag we used for technical debt (I remember we have several but I don't remember which was the correct one 😅 )

Reply:
I've created it [here](https://issues.liferay.com/browse/LPS-158329).